### DISCUSSION TITLE
슬랙 알람 로직 controller 에서 하려고 domain 반환되는 부분

### DISCUSSION THREAD

**Discussion by her0807** [woowacourse-teams/2022-ternoko#191] [Q&A]
service 로 옮기면 안됩니까 ?

**Comment by her0807** [comment]
@HyeonbinSa

**Reply by HyeonbinSa** [reply]
@her0807 
이전 Slack Alarm 기능구현 PR에서 말씀해주신 내용 관련해서 대화를 나눈 적이 있는데 한번 읽어보시면 좋을 것 같습니다.
[슬랙 알람 PR](https://github.com/woowacourse-teams/2022-ternoko/pull/113#discussion_r932830216)
이 리뷰를 받으면서 저도 한번 생각해보는 계기가 되었는데요. 앤지가 트랜잭션과 연계해서 잘 설명해주셔서 왜 컨트롤러에서 사용했는가에 대해서 다시 한번 생각해봤던 리뷰였습니다.

이 부분을 생각해봤을 때 컨트롤러에서 유지하되 Controller에서 Domain을 아는 것에 대해 문제가 있다면 Service 함수의 반환 값을 DTO로 감싸는 방법은 어떠신가요??
